.travis.yml

```diff
@@ -2,23 +2,36 @@ language: python
 python:
   - 2.7
   - 3.6
+
 sudo: false
 
 env:
-    - TEST_DIR="tests"
+  global:
+    - PYPI_PY=3.6
 
-# Setup anaconda
-before_install:
-  - pip install --upgrade pip;
-  - pip install -r requirements.txt
+  matrix:
+    - TEST_DIR="tests"
 
 install:
-  # - conda install --yes pip python=$TRAVIS_PYTHON_VERSION numpy scipy matplotlib ipywidgets
-  - export PATH=$PWD/simpeg:$PATH
+  - pip install --upgrade pip;
+  - pip install twine;
+  - cd gpgLabs; python setup.py install; cd ..
 
 # Run test
 script:
   - nosetests $TEST_DIR -v -s
 
+# deploy to pypi
+after_success:
+  - if [ "$TRAVIS_TAG" ]; then
```

Comment:
a new version of the code needs to be committed and a github release completed for the code to auto-deploy
```
coda patch
```
(https://github.com/aranzgeo/coda) 
cc @thast